### ISSUE TITLE
Create a custom theme and ad a logo

### ISSUE THREAD

**Issue by ccamara** [WarwickRUG/wrug_website#3]
We might want to customise our look and feel, at least to change default colours and adding our nice logo.

**Comment by hturner** [comment]
Whilst we decided to keep the Warwick name (as most folk are indeed from Warwick uni and it defines our area with some separation from the Birmingham group), we do want to be inclusive of people from outside the uni. Therefore I would avoid Warwick corporate branding. You should be able to pick up images and logos used before from Twitter/Meetup - let me know if you have any issues.

**Comment by ccamara** [comment]
Thanks @hturner ! Great explanation. Understood.

Do you happen to have the original logo's vector? (I'd like to create a white version with transparent background -I could easily do that with gimp, but the quality may not be the same)

**Comment by ccamara** [comment]
@all-contributors, could you please add @hturner for questions?

**Comment by ccamara** [comment]
@all-contributors please add @hturner for question

**Comment by Selbosh** [comment]
> I don't, but @Selbosh might.

Yes, I have the originals. Where would you like me to put them or how shall I send them to you?

**Comment by hturner** [comment]
As the original files won't be used directly on the website and could be useful elsewhere, I created a new (private) assets repo where we can put things like these.

**Comment by ccamara** [comment]
thanks @hturner ! @Selbosh , could you please add the sources there? 
I edited the logo in [8d59446](https://github.com/WarwickRUG/wrug_website/commit/8d59446561914c7d25c7bfaef997c9a0ec8a0471) but it is less than optimal (the good news is that the altered one is quite small and doesn't look as bad)
